src/pages/Cart/Cart.js

```diff
@@ -4,45 +4,103 @@ import Footer from '../../components/Footer/Footer';
 import CartLeft from './CartLeft/CartLeft';
 import CartRight from './CartRight/CartRight';
 import './Cart.scss';
+
 class Cart extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            number: 1,
-            allPrice : 13000,
-            price: 13000
+            cartList: [],
+            cartQuantity: "",
         }
     }
     
-    handleDecrease = () => {
-        const { number } = this.state;
-        console.log("Cart에 넘어온 감소버튼 name: ",number);
-        if(number === 1) { 
-            return;
-        } else if(number > 1) {
-            this.setState({
-                number: number - 1,
-            }, () => this.calc())
-        }
+    componentDidMount() {
+        fetch("http://10.58.2.176:8000/order/cart" , {
+            method: "GET",
+            headers: {
+                "Content-type": "application/json",
+                "Authorization" : "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJjdXN0b21lcl9pZCI6MX0.ibPkQgVjNLXv3uLogTIOdjK9A87qVm62YGyhDUJIKm8"
+            },
+        })
+        .then(res => res.json())
+        .then(res => this.setState({
+            cartList : res.cart_items,
+        }, () => console.log(this.state.cartList)))
+    }
+
+    handleDecrease = (list) => {
+        const { cartList } = this.state;
+        const ListCart = [...cartList];
+        const i = ListCart.indexOf(list);
+        cartList[i] = {...list}
+        cartList[i].quantity--;
```

Comment:
이 부분은 한 줄로 합치실 수 있을 것 같습니다.
```suggestion
        cartList[i] = {...list, quantity: --list.quantity}
```
--를 앞에 써준 이유는 의도하신대로 먼저 1을 뺀 값을 저장하기 위해서 입니다.

Reply:
완료했습니다!